app/services/submission/BuildCustomerIdentificationBlock.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services.submission
+
+import models.api._
+import play.api.libs.json.{JsObject, Json}
+import repositories.RegistrationMongoRepository
+import services.submission.JsonUtils.{jsonObject, optional}
+import uk.gov.hmrc.http.InternalServerException
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class BuildCustomerIdentificationBlock(registrationMongoRepository: RegistrationMongoRepository)(implicit ec: ExecutionContext) {
```

Comment:
CustomerIdentificationBlockBuilder sounds better and fits the way we name stuff like this more